README.md

```diff
@@ -86,53 +110,710 @@ ubuntu@infra1:~$ maas-anvil cluster list
 ```
 
-This allows passing a new manifest file with `--manifest` for updating configuration options. If `--manifest -` is passed, then the manifest is loaded from stdin.
+### Join new nodes to the cluster
 
-#### Juju permission denied
+Join the cluster on the joining nodes using the `cluster join` command, the join token that was just created and the roles you want for the specific node.
 
-If you get an error message such as:
+```bash
+ubuntu@infra2:~$ maas-anvil cluster join \
+    --role database --role region --role agent --role haproxy \
+    --token eyJuYW1lIjoibWFhcy00Lm1hYXMiLCJzZWNyZXQiOiI3MmE512342abcdEASWWxOWNlYWNkYmJjMWRmMjk4OThkYWFkYzQzMDAzZjk4NmRkZDI2MWRhYWVkZTIxIiwiZmluZ2VycHJpbnQiOiJlODU5ZmY5NjAwMDU4OGFjZmQ5ZDM0NjFhMDk5NmU1YTU3YjhjN2Q2ZjE4M2NjZDRlOTg2NGRkZjQ3NWMwZWM1Iiwiam9pbl9hZGRyZXNzZXMiOlsiMTAuMjAuMC43OjcwMDAiLCIxMC4yMC4wLjg6NzAwMCJdfQ==
+```
 
 ```bash
-please enter password for $node on anvil-controller:
+ubuntu@infra3:~$ maas-anvil cluster join \
+    --role database --role region --role agent --role haproxy \
+    --token eyJuYW1lIjoibWFhcy00Lm1hYXMiLCJzZWNyZXQiOiI3MmE512342abcdEASWWxOWNlYWNkYmJjMWRmMjk4OThkYWFkYzQzMDAzZjk4NmRkZDI2MWRhYWVkZTIxIiwiZmluZ2VycHJpbnQiOiJlODU5ZmY5NjAwMDU4OGFjZmQ5ZDM0NjFhMDk5NmU1YTU3YjhjN2Q2ZjE4M2NjZDRlOTg2NGRkZjQ3NWMwZWM1Iiwiam9pbl9hZGRyZXNzZXMiOlsiMTAuMjAuMC43OjcwMDAiLCIxMC4yMC4wLjg6NzAwMCJdfQ==
 ```
 
-It is because Juju oauth macaroons typically expire after 24h.
-If you need to interact with the MAAS-anvil Juju controller after this time has passed, you will need to re-authenticate your session.
+## Log into the Juju controller
+
+If you receive an error message like the following:
+
+```
+Please enter password for $node on anvil-controller
+```
 
-You can do this directly using the MAAS-anvil command:
+It is because Juju OAuth macaroons typically expire after 24h. If you need to interact with the MAAS Anvil Juju controller once the macaroon expires, you will need to re-authenticate your session. You can re-authenticate your session with the following command:
 
 ```bash
 ubuntu@$node:~$ maas-anvil juju-login
 ```
 
-You can also manually fetch the login credentials from anvil with:
+You can also manually fetch the login credentials from MAAS Anvil with:
 
 ```bash
 ubuntu@$node:~$ cat ~/snap/maas-anvil/current/account.yaml
 password: $password
 user: $user
 ```
 
-And `juju login` as usual.
+And use `juju login` as usual.
+
+## Configure your MAAS Anvil deployment
+
+When deploying MAAS in high availability, you may need to configure the maximum connection to the database, the virtual IP, TSL, the charms versions used or even the way a component is deployed. MAAS Anvil allows you to configure all of these things, and this section explains how to do it.
```

Comment:
TLS, not TSL

Reply:
Ahh dyslexia has struck again, good catch, thanks!